src/unread/__tests__/unreadHuddlesReducer-test.js

```diff
@@ -115,20 +120,16 @@ describe('unreadHuddlesReducer', () => {
     test('if message is sent by self, do not mutate state', () => {
       const initialState = deepFreeze([]);
 
+      const message2 = eg.pmMessage({
+        sender: eg.selfUser,
+        recipients: [eg.makeUser({ name: 'john' }), eg.makeUser({ name: 'mark' }), eg.selfUser],
```

Comment:
These are fine, but I usually just turn them into `eg.otherUser` and `eg.thirdUser`, if there don't appear to be other places that depend on the specific names.

(If there are such places, usually it's ideal to replace those too -- it just makes it more work and so I don't always.)

Reply:
Would that be OK to do in the same commit?

Reply:
Yeah, or can break them up if the commit is getting too unwieldy.